### ISSUE TITLE
Additional useful shortcuts

### ISSUE THREAD

**Issue by jake8655** [hyprwm/hyprlock#457]
### Description

I always use `<C-m>` in the terminal instead of `Enter` and `<C-h>` instead of `Backspace` so its a habit of mine. It would be nice if I could use them in hyprlock as well.

### Note
I can open a quick PR if you say its good to go.

**Comment by jake8655** [comment]
Sure, I could look into it, I just didnt want to open a PR without someone saying it's a good idea. I have made the change in a fork for now and am using that, but if it can get into upstream that would be great!

**Comment by LorenzHohermuth** [comment]
No, I wanted to try my hands on this project and thought it was a great first issue.
I want to try to add it to the config for a more universal solution.
